Runtime/Components/CinemachineSplineDolly.cs

```diff
@@ -0,0 +1,399 @@
+            + "to the path, Y is up, and Z is parallel to the path.")]
+        public Vector3 m_PathOffset = Vector3.zero;
+
+        /// <summary>How aggressively the camera tries to maintain the offset along the x, y, or z directions in path local space.
+        /// Meaning:
+        /// - x represents the axis that is perpendicular to the path.
+        /// - y represents the axis that is defined by the path-local up direction.
+        /// - z represents the axis that is parallel to the path.
+        /// Smaller numbers are more responsive. Larger numbers give a more heavy, slowly responding camera.
+        /// Using different settings per axis can yield a wide range of camera behaviors.</summary>
+        [Tooltip("How aggressively the camera tries to maintain the offset along the x, y, or z directions in path local space. \n" +
+            "- x represents the axis that is perpendicular to the path. \n" +
+            "- y represents the axis that is defined by the path-local up direction. \n" +
+            "- z represents the axis that is parallel to the path. \n" +
+            "Smaller numbers are more responsive, larger numbers give a more heavy, slowly responding camera. " +
+            "Using different settings per axis can yield a wide range of camera behaviors.")]
+        public Vector3 m_Damping = Vector3.zero;
+
+        /// <summary>Different ways to set the camera's up vector</summary>
+        [DocumentationSorting(DocumentationSortingAttribute.Level.UserRef)]
+        public enum CameraUpMode
+        {
+            /// <summary>Leave the camera's up vector alone.  It will be set according to the Brain's WorldUp.</summary>
+            Default,
+            /// <summary>Take the up vector from the path's up vector at the current point</summary>
+            Path,
+            /// <summary>Take the up vector from the path's up vector at the current point, but with the roll zeroed out</summary>
+            PathNoRoll,
+            /// <summary>Take the up vector from the Follow target's up vector</summary>
+            FollowTarget,
+            /// <summary>Take the up vector from the Follow target's up vector, but with the roll zeroed out</summary>
+            FollowTargetNoRoll,
+        };
+
+        /// <summary>How to set the virtual camera's Up vector.  This will affect the screen composition.</summary>
+        [Tooltip("How to set the virtual camera's Up vector.  This will affect the screen composition, because "
+            + "the camera Aim behaviours will always try to respect the Up direction.")]
+        public CameraUpMode m_CameraUp = CameraUpMode.Default;
+
+        /// <summary>
+        /// How aggressively the camera tries to track the target rotation (x: Pitch, y: Yaw, z: Roll).
+        /// Smaller numbers are more responsive. Larger numbers give a more heavy, slowly responding camera.</summary>
+        /// </summary>
+        public Vector3 m_RotationDamping = Vector3.zero;
```

Comment:
Would it make more sense for this to be a single float value, used to damp the slerp from old rotation to the new one?  Why do we need component-wise damping here?  See CinemachineTransposer,cs line 267 for implementation example.

Reply:
I copied it from Tracked dolly, but I agree that just one rotation damping is enough. We discussed it with Swap too.